docs/issues.md

```diff
@@ -22,6 +22,8 @@ title: Common Issues
 - MathJax defines `\color` to be like `\textcolor` by default; set KaTeX's
   `colorIsTextColor` option to `true` for this behavior.  KaTeX's default
   behavior matches MathJax with its `color.js` extension enabled.
+- Equivalents of MathJax `\class`, `\cssId`, and `\style` are `\htmlClass`,
+  `\htmlId`, and `\htmlStyle`, respectively, to avoid ambiguity.
```

Comment:
Thanks for adding this.  If people want to use the MathJax variants they can set up some macros.